pkg/apicheck/check.go

```diff
@@ -130,35 +129,35 @@ func (c *ApiConnectivityCheck) getWorkerPeersResponse() peers.Response {
 	}
 
 	c.config.Log.Info("Error count exceeds threshold, trying to ask other nodes if I'm healthy")
-	nodesToAsk := c.config.Peers.GetPeersAddresses(peers.Worker)
-	if nodesToAsk == nil || len(nodesToAsk) == 0 {
+	peersToAsk := c.config.Peers.GetPeersAddresses(peers.Worker)
+	if peersToAsk == nil || len(peersToAsk) == 0 {
 		c.config.Log.Info("Peers list is empty and / or couldn't be retrieved from server, nothing we can do, so consider the node being healthy")
-		//todo maybe we need to check if this happens too much and reboot
+		// TODO: maybe we need to check if this happens too much and reboot
 		return peers.Response{IsHealthy: true, Reason: peers.HealthyBecauseNoPeersWereFound}
 	}
 
 	apiErrorsResponsesSum := 0
-	nrAllNodes := len(nodesToAsk)
-	// nodesToAsk is being reduced in every iteration, iterate until no nodes left to ask
-	for i := 0; len(nodesToAsk) > 0; i++ {
+	nrAllPeers := len(peersToAsk)
+	// peersToAsk is being reduced at every iteration, iterate until no peers left to ask
+	for i := 0; len(peersToAsk) > 0; i++ {
 
-		// start asking a few nodes only in first iteration to cover the case we get a healthy / unhealthy result
+		// start asking a few peers only in first iteration to cover the case we get a healthy / unhealthy result
 		nodesBatchCount := reboot.MinNodesNumberInBatch
 		if i > 0 {
 			// after that ask 10% of the cluster each time to check the api problem case
-			nodesBatchCount = len(nodesToAsk) / reboot.MaxBatchesAfterFirst
+			nodesBatchCount = len(peersToAsk) / reboot.MaxBatchesAfterFirst
 			if nodesBatchCount < reboot.MinNodesNumberInBatch {
 				nodesBatchCount = reboot.MinNodesNumberInBatch
 			}
 		}
 
 		// but do not ask more than we have
-		if len(nodesToAsk) < nodesBatchCount {
-			nodesBatchCount = len(nodesToAsk)
+		if len(peersToAsk) < nodesBatchCount {
+			nodesBatchCount = len(peersToAsk)
 		}
 
-		chosenNodesAddresses := c.popNodes(&nodesToAsk, nodesBatchCount)
-		healthyResponses, unhealthyResponses, apiErrorsResponses, _ := c.getHealthStatusFromPeers(chosenNodesAddresses)
+		chosenPodIPs := c.popPeerIPs(&peersToAsk, nodesBatchCount)
```

Comment:
nit: at other places we use `peer` instead of `node` or `pod`, what about naming this var `chosenPeerIPs` as well?

Reply:
you're right, just a typo here